src/components/app/header/index.tsx

```diff
@@ -49,6 +50,19 @@ const Header: FunctionComponent = () => {
     case !subscriber && !loadingSubscriber:
       ActiveCTA = () => <OnlinePresenceCTA variant="header" />
       break
+    case !viewer?.is_pro:
```

Comment:
if they aren't pro and don't have any other options, let's just prompt them to join